src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -16,6 +16,7 @@
 from os import urandom
 import datetime
 import json
+import random
```

Comment:
is this used somewhere?

Reply:
yes, used to generate a random number for the cert name